sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -400,6 +401,13 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper wit
     }
   }
 
+  private def findGroupColumn(alias: Alias): Option[AttributeReference] = alias match {
```

Comment:
Per `SortOrder(attr: AttributeReference...`, it's always `AttributeReference`. Should it address `Alias`?

Reply:
Yes. If users specify an `Alias` for group column. It will be `Alias(alias: Alias, _)`.